charts/kuadrant-operator/Chart.yaml

```diff
@@ -44,3 +44,102 @@ maintainers:
     name: Jim Fitzpatrick
   - email: chfan@redhat.com
     name: Kevin Chi Fan
+annotations:
+  artifacthub.io/category: networking
+  artifacthub.io/crds: |
+    - kind: Kuadrant
+      version: v1
```

Comment:
@maleck13 should Kuadrant CRD be promoted to v1 too?

Reply:
👎